src/dialogue.cpp

```diff
@@ -91,10 +91,10 @@ void Dialogue::Advance() {
 
 // Resets the conversation, e.g. the protag talks to the NPC again.
 void Dialogue::Reset() {
-    //TODO
+    //TODO - implement
 }
 
 // Verifies if the graph is cycle-free. Returns true on success.
 bool Dialogue::CheckCycle() {
-    //TODO
+    //TODO - implement
```

Comment:
Don't communicate with _commits_ & TODO, unless necessary. 
This creates conflicts in the comments, so not that fatal, but people still have to resolve them.